pkg/config/containers.conf

```diff
@@ -458,8 +458,11 @@ default_sysctls = [
 #            "/usr/bin/kata-fc",
 # ]
 
-# The [engine.runtimes] table MUST be the last entry in this file.
+[engine.volume_plugins]
+# testplugin = "/run/podman/plugins/test.sock"
+
+# The [engine.volume_plugins] table MUST be the last entry in this file.
 # (Unless another table is added)
 # TOML does not provide a way to end a table other than a further table being
-# defined, so every key hereafter will be part of [runtimes] and not the main
-# config.
+# defined, so every key hereafter will be part of [volume_plugins] and not the
+# main config.
```

Comment:
Have we done away with runtimes?

Reply:
We have a new table at the bottom, `volume_plugins`. The last item in the config must be a table. It does not necessarily need to be the runtimes table.

Reply:
OK, this was my confusion point.  The only mention I see in the man page to "the runtimes table" is under `runtime` as a pointer to it.  Are the "runtime_supports_*" considered to be the runtimes table(s)?  Regardless, this line in the doc needs further help:

```
runtime="crun"

Default OCI specific runtime in runtimes that will be used by default. Must refer to a member of the runtimes table.
```

There's not bolded "runtimes" table and the `runtime_supports_*` entries all purport to be lists.  So are we missing the runtimes table?

Reply:
FWIW, I'm looking solely at the md file here, not the conf itself.

Reply:
The `runtime_supports` arrays (not tables, tables are maps) refer to specific features of specific runtimes. The runtimes table is the bottom of the config file until this patch is merged, and it seems like you are correct that it's entirely missing from the manpage.

Reply:
I'm not seeing changes for the missing runtimes table being added here.  Can you add it or at least spin up an issue for it to be done later please?